functions/v2/deploy-function/deploy_function.py

```diff
@@ -0,0 +1,68 @@
+# [START create_cloud_function_v2]
```

Comment:
This is also an invalid region tag; I'm not sure the intent of this sample and where it should go.

Reply:
cc. @m-strzelczyk